src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.cs

```diff
@@ -54,11 +54,14 @@ internal abstract partial class Layouter : ILayouter
 
 		internal Size _unclippedDesiredSize;
 
+		private UIElement _elementAsUIElement;
```

Comment:
property here instead of field?

Reply:
It's `private` ... why use a property?

Reply:
For legibility, since it will always point to the same instance as Panel. Why use a field?

Reply:
For performance, in that case :)